printer_mods/jatson/plug_panel_schurter/README.md

```diff
@@ -0,0 +1,13 @@
+# Plug Panel Mod for Schurter CD44.1101.151
+Connection panel for Voron 2.4 with Schurter CD44.1101.151 power entry modul.
```

Comment:
Im guessing you meant module here instead of modul :)

Reply:
typo fixed